packages/salesforcedx-vscode-core/src/orgPicker/orgList.ts

```diff
@@ -43,7 +43,7 @@ export class OrgList implements vscode.Disposable {
   }
 
   public displayDefaultUsername(defaultUsernameorAlias?: string) {
-    if (!defaultUsernameorAlias) {
+    if (defaultUsernameorAlias) {
       this.statusBarItem.text = `$(plug) ${defaultUsernameorAlias}`;
```

Comment:
What is `plug`?
Where is it coming from?
When I run defaultUsernameorAlias is "test1-org", and this.statusBarItem.text is resolving to "'$(plug) test1-org'"